### PR TITLE
First working version of app

### PR DESCRIPTION
**Pull request by balthazzar**
chess start logic, twitter acc access, chess engine, chess engine api,
move chess logic from titter

### CODE REVIEW

**Comment by AlexeyAnshakov** [comment]
@michbil please review

**Comment by AlexeyAnshakov** [comment]
@michbil please review

File: appData/utils.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ var nconf = require('../wrio_nconf.js');
 var MongoClient = require('mongodb')
 	.MongoClient;
 
-var url = 'mongodb://' + nconf.get('mongo:user') + ':' + nconf.get('mongo:password') + '@' + nconf.get('mongo:host') + '/' + nconf.get('mongo:dbname');
+var url = 'mongodb://' + nconf.get('db:user') + ':' + nconf.get('db:password') + '@' + nconf.get('db:host') + '/' + nconf.get('db:dbname');
```
Comment (michbil) [inline appData/utils.js]: @balthazzar Now we are using mongo settings record {} config format in all WRIO apps, so better to keep config pattern uniform for all apps.

**Comment by AlexeyAnshakov** [comment]
@michbil can we merge it?

**Comment by michbil** [comment]
No, double scrollbars on titter widget appear again.

**Comment by michbil** [comment]
Need to find more elegant solution

**Comment by michbil** [comment]
PR ok

**Pull request merged**